src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -6,21 +6,21 @@ import FormControl from '../form-control/form-control.jsx';
 import Icon from '../icon/icon.jsx';
 import styles from './custom-field-input-text.css';
 
-function isInvalid(error, readOnly) {
-  return error && !readOnly;
-}
-
 export default function CustomFieldInputText(props) {
   const defaultRef = useRef(null);
   const [validationMessage, setValidationMessage] = useState('');
 
   const inputRef = props.inputRef || defaultRef;
   const labelId = `${props.id}-label`;
 
+  const isInvalid = () => {
```

Comment:
Good move! I'm not sure why we need to check `readOnly` though, couldn't we assume that the existing value is valid?

Reply:
This was just consolidating / refactoring the previous logic. I am not sure why it was like this.